control/tests/statesp_test.py

```diff
@@ -561,6 +561,69 @@ def test_str(self):
         assert str(sysdt1) == tref + "\ndt = 1.0\n"
 
 
+    def test_latex_repr(self):
+        # Test '_latex_repr_' with different config values
+        # This is a 'gold image' test, so if you change behaviour,
+        # you'll need to regenerate the reference results.
+        # Try something like:
+        #   control.reset_defaults()
+        #   print(f'g1_p3_p = {g1._repr_latex_()!r}')
+        #   control.set_defaults('statesp', latex_num_format='.5g')
+        #   print(f'g1_p5_p = {g1._repr_latex_()!r}')
+        #   control.reset_defaults()
+        #   control.set_defaults('statesp', latex_repr_type='separate')
+        #   print(f'g1_p3_s = {g1._repr_latex_()!r}')
+        #   control.set_defaults('statesp', latex_num_format='.5g')
+        #   print(f'g1_p5_s = {g1._repr_latex_()!r}')
+        from control import set_defaults, reset_defaults
+        g1 = StateSpace([[np.pi, 1e100],[-1.23456789, 5e-23]],
+                        [[0], [1]],
+                        [[987654321, 0.001234]],
+                        [[5]])
+        g2 = StateSpace([],
+                        [],
+                        [],
+                        [[1.2345,-2e-200],[-1,0]])
+
+        g1_p3_p = '\\[\n\\left(\n\\begin{array}{rllrll|rll}\n3.&\\hspace{-1em}14&\\hspace{-1em}\\phantom{\\cdot}&1\\phantom{.}&\\hspace{-1em}&\\hspace{-1em}\\cdot10^{100}&0\\phantom{.}&\\hspace{-1em}&\\hspace{-1em}\\phantom{\\cdot}\\\\\n-1.&\\hspace{-1em}23&\\hspace{-1em}\\phantom{\\cdot}&5\\phantom{.}&\\hspace{-1em}&\\hspace{-1em}\\cdot10^{-23}&1\\phantom{.}&\\hspace{-1em}&\\hspace{-1em}\\phantom{\\cdot}\\\\\n\\hline\n9.&\\hspace{-1em}88&\\hspace{-1em}\\cdot10^{8}&0.&\\hspace{-1em}00123&\\hspace{-1em}\\phantom{\\cdot}&5\\phantom{.}&\\hspace{-1em}&\\hspace{-1em}\\phantom{\\cdot}\\\\\n\\end{array}\\right)\n\\]'
```

Comment:
maybe reformat as `r''` strings for better readability?

Reply:
These were generated.  Can one coax `repr` (or whatever) to spit out raw strings?